### ISSUE TITLE
length_scale seems to be ignored

### ISSUE THREAD

**Issue by mmaelicke** [GeoStat-Framework/GSTools#45]
Hey Guys,

One of my codes that uses your lib to generate `SRF` started producing weird results. I tried to identify the problem and to me it looks like recently `length_scale` of the used `CovModel` is not reflected in the final field, anymore.

I am running gstools version 1.1.1 and recently upgraded numpy to 1.17, which might be related.

A minimal example that should illustrate the problem:

```python
x = y = range(100)
model = gstools.Gaussian(dim=2, length_scale=1., var=1)
srf = gstools.SRF(model)
field = srf((x, y), mesh_type='structured')
srf.plot()
```
gives:
![image](https://user-images.githubusercontent.com/2826034/70372842-6daebe00-18e4-11ea-9847-40da326123f9.png)

while

```python
x = y = range(100)
model = gstools.Gaussian(dim=2, length_scale=70., var=1)
srf = gstools.SRF(model)
field2 = srf((x, y), mesh_type='structured')
srf.plot()
```
gives: 

![image](https://user-images.githubusercontent.com/2826034/70372848-8028f780-18e4-11ea-9eee-1848260c9742.png)

shouldn't these two fields have significantly different correlation lengths? Or am I missing something here? 
Thanks!

**Comment by mmaelicke** [comment]
Sorry.

It should be `len_scale` and not `length_scale`. Then everything works as expected.
But, why did I not run into an `AttributeError`?
Never mind,
Best.

**Comment by MuellerSeb** [comment]
Hey @mmaelicke ,
You code created an optional argument:

```python
In [8]: model.arg                                                                                        
Out[8]: ['var', 'len_scale', 'nugget', 'anis', 'angles', 'length_scale']
In [9]: model.opt_arg                                                                                    
Out[9]: ['length_scale']
In [10]: model.length_scale                                                                              
Out[10]: 70.0
```

which is not used in your model. This reflacts the fact, that you can create your own model with what ever number of optional arguments.

But you are right, that there should be at least a warning, when a unknown parameter is given when the model is created.

I will have a look at that. Thanks for your report!
Cheers, Sebastian

**Comment by MuellerSeb** [comment]
Solved by #46
